docs/modules/indexes/document_loaders/examples/acreom.ipynb

```diff
@@ -0,0 +1,75 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e310c8dc-acd0-48d2-801c-f37ce99acd2d",
+   "metadata": {},
+   "source": [
+    "# Loading documents from acreom vault"
```

Comment:
Nit: please title with `# acreom` (make it compatible w/ documentation)

Reply:
fixed